frontend/src/app/plot/plot.component.ts

```diff
@@ -30,64 +30,74 @@ import { UploadService } from '../upload.service'
 export class PlotComponent implements OnInit {
   /**
    * plot_data is an array of maps that define what is plotted.
-   * It is initialized with (0,0) point so that the plot appears when the page is opened.
+   * It is initialized with a (0,0) point so that the plot appears when the page is opened.
    * Each map defines a single plotly trace with possible options.
    * Trace options:
    *    x: Takes an array of x-axis values that correspond to the y-axis.
    *        Usually timestamps.
    *    y: Takes an array of y-axis values.
    *    (Note: The lengths of x and y must be equal.)
    *    type: How the data is displayed to the user. 'scattergl' and 'historgram'
-   *      will most likely be the only types used. (Avoid 'scatter' type since is
-   *      has poorer performance on large datasets than 'scattergl'.) 
+   *      will most likely be the only types used. (Avoid 'scatter' type since it
+   *      has poorer performance on large datasets than 'scattergl'.)
    *    mode: Finer details for how to display the type. 'markers' displays only a dot
    *      for each datapoint. 'lines' displays a line through all datapoints. 'lines+markers'
    *      displays both.
    */
-  plot_data = [{x: [0], y: [0], type: 'scattergl',
-   mode: 'markers', id: 0, visible: 'true',
-   name: 'Placeholder Point'
-  }]
+  plot_data = [
+    {
+      x: [0],
+      y: [0],
+      type: 'scattergl',
+      mode: 'markers',
+      id: 0,
```

Comment:
You could align the values.
```
x:          [0]
y:          [0]
type:       'scattergl'
```

Reply:
The linter tells me not to do this, I agree it would look way better to align them. Should I ignore the linter this time?

Reply:
lets follow the linter!